config/config.go

```diff
@@ -578,6 +588,13 @@ func New(v *viper.Viper) (*Configuration, error) {
 		return nil, err
 	}
 
+	// Update account defaults and generate base json for patch
+	c.AccountDefaults.CacheTTL = c.CacheURL.DefaultTTLs // comment this out to set explicitly in config
+	var err error
+	if c.AccountDefaultsJSON, err = json.Marshal(c.AccountDefaults); err != nil {
```

Comment:
If we're always gonna construct the `AccountDefaultsJSON` from the `AccountDefaults` struct then former should likely _not_ be a config option. I can only see a reason for it being a config option if we want the host to be able to provide the account defaults as a json string. They can do that right now as `AccountDefaultsJSON` is a config option but we always override it

Reply:
I can look into alternatives if this is really a deal breaker, but I want to point out that I followed the same implementation done a few lines above for [GDPR.NonStandardPublisherMap](https://github.com/prebid/prebid-server/blob/master/config/config.go#L586-L591), [BlacklistedAcctMap](https://github.com/prebid/prebid-server/blob/master/config/config.go#L600-L605) and [BlacklistedApps](https://github.com/prebid/prebid-server/blob/master/config/config.go#L593-L598) which were accepted about a year ago.

Reply:
Added commit dc8a74d to make the json a private config member, exposed to other packages via accessor method.